server/models/Group.js

```diff
@@ -394,15 +398,24 @@ module.exports = function(Sequelize, DataTypes) {
           }
         })
         .then(userGroup => Promise.resolve(!!userGroup));
+      },
+
+      getSuperCollectiveData() {
```

Comment:
I'm really questioning the need to introduce a new concept for "super collective". All collectives should be "super collectives" (because they are all super, ah ah)

E.g. MochaJS could link to other collectives that are dependencies.